base/client.jl

```diff
@@ -236,6 +236,20 @@ function exec_options(opts)
     global have_color     = colored_text(opts)
     global is_interactive = (opts.isinteractive != 0)
 
+    # Enable verbose debugging options when requested by other frameworks
+    debug_env_vars = (
+        "RUNNER_DEBUG",   # github actions when UI "debug logging" is enabled
+        "CI_DEBUG_TRACE", # gitlab CI when UI "debug" toggle is enabled
+        "SYSTEM_DEBUG",   # azure pipelines when UI "System diagnostics" is enabled
+    )
+    for v in debug_env_vars
+        if get_bool_env(v, false)
+            Base.TRACE_EVAL = :loc # Enable --trace-eval (location only)
```

Comment:
If it’s already `:full` I guess we shouldn’t downgrade it?

Reply:
```suggestion
            Base.TRACE_EVAL = Base.TRACE_EVAL === :full ? :full : :loc # Enable --trace-eval (location only)
```